README.md

```diff
@@ -1,7 +1,7 @@
 # Discourse Clicable Topic Component
 This component makes the entire area of a `topic-list-item` clickable, as opposed to just the title.
 
-![Discourse Clickable Topic Preivew](readme-assets/preview.gif)
+![Discourse Clickable Topic Preivew](https://github.com/discourse/discourse-clickable-topic/assets/17474474/720acb71-b283-4a3a-ad98-39a0d502ca1e)
 
 [Preview on theme-creator](https://theme-creator.discourse.org/theme/jordan.vidrine/clickable-topic 
-)
\ No newline at end of file
+)
```

Comment:
Why is this on a new line? This should be at the end of the previous one, right?